### PR TITLE
Add --tls-verify, --cert-dir, and --quiet flags to kpod pull

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
LGTM

File: cmd/kpod/pull.go [hunk 1]
```diff
@@ -17,17 +18,29 @@ import (
...
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.BoolTFlag{
+			Name:  "tls-verify",
+			Usage: "require HTTPS and verify certificates when contacting registries (default: true)",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "Suppress output information when pulling images",
+		},
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/pull.go]: nit, move quite up before signature please

File: completions/bash/kpod [hunk 2]
```diff
@@ -172,10 +172,14 @@ _kpod_pull() {
     local options_with_args="
     --authfile
     --creds
+    --cert-dir
     --signature-policy
     "
     local boolean_options="
     --all-tags -a
+    --tls-verify
```
Comment (TomSweeneyRedHat) [inline completions/bash/kpod]: nit, drop this line below the next two.

File: docs/kpod-pull.1.md [hunk 3]
```diff
@@ -68,6 +72,14 @@ Pathname of a signature policy file to use.  It is not recommended that this
 option be used, as the default behavior of using the system-wide default policy
 (frequently */etc/containers/policy.json*) is most often preferred
 
+**--tls-verify**
+
+Require HTTPS and verify certificates when contacting registries (default: true)
+
+**--quiet, -q**
```
Comment (TomSweeneyRedHat) [inline docs/kpod-pull.1.md]: alphabetize here too please. quiet before tls.

File: docs/kpod-pull.1.md [hunk 4]
```diff
@@ -95,7 +107,18 @@ Storing signatures
...
+Copying blob sha256:6d987f6f42797d81a318c40d442369ba3dc124883a0964d40b0c8f4f7561d913
+ 1.90 MB / 1.90 MB [========================================================] 0s
+Copying config sha256:ad4686094d8f0186ec8249fc4917b71faa2c1030d7b5a025c29f26e19d95c156
+ 1.41 KB / 1.41 KB [========================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+```
+# kpod pull --tls-verify false --cert-dir image/certs docker.io/umohnani/finaltest
```
Comment (TomSweeneyRedHat) [inline docs/kpod-pull.1.md]: double check, but I think you need = before tls-verify.

**Comment by TomSweeneyRedHat** [comment]
LGTM overall, a few ordering nits of nits.  I'm assuming you and @mtrmac are tied off with his notes from the previous submit?

**Comment by mtrmac** [comment]
> I'm assuming you and @mtrmac are tied off with his notes from the previous submit?

Those have nothing to do with the new `--tls-verify`/`--cert-dir`/`--quiet` options, I don’t think they should block this PR.

File: cmd/kpod/pull.go (hunk 1 above)
Comment (umohnani8) [reply cmd/kpod/pull.go]: fixed

File: completions/bash/kpod (hunk 2 above)
Comment (umohnani8) [reply completions/bash/kpod]: fixed

File: docs/kpod-pull.1.md (hunk 3 above)
Comment (umohnani8) [reply docs/kpod-pull.1.md]: fixed

File: docs/kpod-pull.1.md (hunk 4 above)
Comment (umohnani8) [reply docs/kpod-pull.1.md]: fixed

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**